src/components/item/sharing/membershipTable/GuestItemMembershipTableRow.tsx

```diff
@@ -1,36 +1,78 @@
-import { TableCell, Typography } from '@mui/material';
+import { TableCell, Tooltip, Typography } from '@mui/material';
 
-import { DiscriminatedItem, ItemMembership } from '@graasp/sdk';
+import {
+  DiscriminatedItem,
+  ItemLoginSchema,
+  ItemLoginSchemaStatus,
+  ItemMembership,
+} from '@graasp/sdk';
 
-import { useEnumsTranslation } from '@/config/i18n';
+import { useBuilderTranslation, useEnumsTranslation } from '@/config/i18n';
+import { hooks } from '@/config/queryClient';
+import { BUILDER } from '@/langs/constants';
 
 import { buildItemMembershipRowId } from '../../../../config/selectors';
 import DeleteItemMembershipButton from './DeleteItemMembershipButton';
 import { StyledTableRow } from './StyledTableRow';
 
+const DISABLED_COLOR = '#a5a5a5';
+
 const GuestItemMembershipTableRow = ({
   data,
   itemId,
+  itemLoginSchema,
 }: {
   data: ItemMembership;
   itemId: DiscriminatedItem['id'];
+  itemLoginSchema?: ItemLoginSchema;
 }): JSX.Element => {
   const { t: translateEnums } = useEnumsTranslation();
+  const { t: translateBuilder } = useBuilderTranslation();
+  const { data: currentAccount } = hooks.useCurrentMember();
+
+  let isDisabled = false;
+  if (
+    currentAccount?.id !== data.account.id &&
+    (!itemLoginSchema ||
+      itemLoginSchema.status === ItemLoginSchemaStatus.Disabled)
+  ) {
+    isDisabled = true;
+  }
```

Comment:
I'm not sure that the if add clarity compared to an inline const. Maybe we should use to consts?

```suggestion
  const itemLoginSchemaIsDisabled = !itemLoginSchema || itemLoginSchema.status === ItemLoginSchemaStatus.Disabled;
  const isDisabled = currentAccount?.id !== data.account.id && itemLoginSchemaIsDisabled;
```